src/Akeneo/Tool/Bundle/MeasureBundle/Resources/public/pages/list/MeasurementFamilyRow.tsx

```diff
@@ -13,7 +13,7 @@ type MeasurementFamilyRowProps = {
 };
 
 const MeasurementFamilyRow = ({measurementFamily}: MeasurementFamilyRowProps) => {
-  const locale = useUserContext().get('uiLocale');
+  const locale = useUserContext().get('catalogLocale');
```

Comment:
```suggestion
  const locale = useUserContext().get('uiLocale');
```
Nope, we display the label in UI locale on purpose in the Measurements settings

Reply:
So, what is described in https://akeneo.atlassian.net/browse/SDS-15985 (step 7) is not a bug ?

Reply:
Saw together, it's not a bug.